### PR TITLE
Updated kodi-9999.ebuild

### PR DESCRIPTION
**Pull request by schorsch1976**
Please konsider to pull my changed for kodi-9999.ebuild. 

- nvidia-driver is not needed anymore for vdpau. I personally use the OSS radeon vdpau with kodi.
- kodi needs gnutls to build

### CODE REVIEW

File: media-tv/kodi/kodi-9999.ebuild [hunk 1]
```diff
@@ -112,8 +112,8 @@ COMMON_DEPEND="${PYTHON_DEPS}
 	vaapi? ( x11-libs/libva[opengl] )
 	vdpau? (
 		x11-libs/libvdpau
-		>=x11-drivers/nvidia-drivers-180.51
 	)
+	net-libs/gnutls
```
Comment (frace) [inline media-tv/kodi/kodi-9999.ebuild]: Please honor the alphabetical order of dependencies.

**Comment by schorsch1976** [comment]
About the vdpau driver i agree that we should check for a suitable driver. How about to check for a suitable driver for the radeon or intel OSS vdpau driver?

From my knowledge it depends on the kernel version and the mesa version. 
See 
https://wiki.archlinux.org/index.php/VDPAU

It is not just as simple as nvidia-drivers. In the open source case the provider is
x11-libs/libva-vdpau-driver
[I] x11-libs/libva-vdpau-driver
     Available versions:  0.7.3{tbz2} ~0.7.4 0.7.4-r1{tbz2} ~0.7.4-r2 **9999 {debug opengl ABI_MIPS="n32 n64 o32" ABI_PPC="32 64" ABI_S390="32 64" ABI_X86="32 64 x32"}
     Installed versions:  0.7.4-r1{tbz2}(21:29:12 16.07.2014)(-debug -opengl ABI_MIPS="-n32 -n64 -o32" ABI_PPC="-32 -64" ABI_S390="-32 -64" ABI_X86="64 -32 -x32")
     Homepage:            http://www.freedesktop.org/wiki/Software/vaapi
     Description:         VDPAU Backend for Video Acceleration (VA) API

[I] x11-libs/libvdpau
     Available versions:  0.7{tbz2} ~0.8 ~0.9 {doc dri test ABI_MIPS="n32 n64 o32" ABI_PPC="32 64" ABI_S390="32 64" ABI_X86="32 64 x32"}
     Installed versions:  0.7{tbz2}(09:16:05 07.06.2014)(dri -doc ABI_MIPS="-n32 -n64 -o32" ABI_X86="64 -32 -x32")
     Homepage:            http://www.freedesktop.org/wiki/Software/VDPAU
     Description:         VDPAU wrapper and trace libraries

[I] x11-misc/vdpauinfo
     Available versions:  0.0.6{tbz2} ~0.1 ~0.9
     Installed versions:  0.0.6{tbz2}(00:44:38 07.05.2014)
     Homepage:            http://www.freedesktop.org/wiki/Software/VDPAU
     Description:         Displays info about your card's VDPAU support

-------
I was not sure if requests and changes are handles over the gentoo bug tracker. So i had not yet created a bug report at gentoo bugzilla.

I have uploaded the requested changes to my repository (ordering and one line vdpau).

**Comment by frace** [comment]
@schorsch1976 No, we don't handle requests and changes for the ***unofficial overlay*** at the official Gentoo bug tracker. It's just fine to report issues related to the *overlay* right here.

At the same time I don't the want the ebuild to diverge too much from the original Gentoo ebuild. So in order to stay in sync I'd like to see the changes we make in the overlay to be merged inside the official Gentoo ebuild. On the other hand we include/backport changes made in the official Gentoo ebuild inside the overlay ebuild. That's why it makes sense to report the issue - at best including a patch - at the Gentoo bug tracker, too.

**Comment by schorsch1976** [comment]
The overview in the arch wiki is the best i can find.
https://wiki.archlinux.org/index.php/VDPAU
http://de.wikipedia.org/wiki/Video_Decode_and_Presentation_API_for_Unix

For an overview on the driver capabilities see
http://www.x.org/wiki/RadeonFeature/

The important row is "Video Decode (VDPAU/OpenMax/VAAPI) on UVD"

I can confirm that vdpau runs on my AMD E-350. I tried to downgrade the kernel until 3.12 and it did still run. In 3.17 a bug in the radeon driver was fixed that caused some crashes, but right now i am unable to find the commit.

For general information about XMBC and vdpau on radeon cards see
http://forum.kodi.tv/showthread.php?tid=174854

**Comment by schorsch1976** [comment]
Oh, i forgot for intel the table is here
http://www.x.org/wiki/IntelGraphicsDriver/

The important rows are VA-API (MPEG) / VA_API (H264). But i can not confirm that vdpau is running on them.

**Comment by frace** [comment]
Picked the addition of gnutls for now. See https://github.com/frace/kodi-overlay/commit/daf78aba43f6248453bd4fd7655f8284e22cdbd8.
I need to find some free time to get into the vdpau topic. Maybe next weekend...

**Comment by frace** [comment]
Fixed with https://github.com/frace/kodi-overlay/commit/5cbe211e7e93d7cbea285c82e22b9bacbe3b474b

**Pull request closed without merging**